spec/spec_helper.rb

```diff
@@ -1,4 +1,4 @@
-#This file is generated by ModuleSync, do not edit.
+# This file is generated by ModuleSync, do not edit.
```

Comment:
Any changes in this file will be overwritten when a module sync is ran, I would suggest making the changes in the modulesync configs: https://github.com/puppetlabs/modulesync_configs/blob/master/moduleroot/spec/spec_helper.rb

Reply:
This change was made to modulesync